<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>title</class>
 <widget class="QWidget" name="title">
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>578</width>
	<height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
	<widget class="QLabel" name="logo">
	 <property name="text">
	  <string/>
	 </property>
	 <property name="pixmap">
	  <pixmap resource="cloud_disk.qrc">:/image/Resources/image/images/logo.ico</pixmap>
	 </property>
	</widget>
   </item>
   <item>
	<widget class="QLabel" name="wgtitle">
	 <property name="text">
	  <string>百度云盘</string>
	 </property>
	</widget>
   </item>
   <item>
	<spacer name="horizontalSpacer">
	 <property name="orientation">
	  <enum>Qt::Horizontal</enum>
	 </property>
	 <property name="sizeHint" stdset="0">
	  <size>
	   <width>304</width>
	   <height>20</height>
	  </size>
	 </property>
	</spacer>
   </item>
   <item>
	<widget class="QToolButton" name="button_set">
	 <property name="text">
	  <string>...</string>
	 </property>
	 <property name="icon">
	  <iconset resource="cloud_disk.qrc">
	   <normaloff>:/image/Resources/image/images/login_setting.png</normaloff>:/image/Resources/image/images/login_setting.png</iconset>
	 </property>
	 <property name="iconSize">
	  <size>
	   <width>32</width>
	   <height>32</height>
	  </size>
	 </property>
	 <property name="autoRaise">
	  <bool>true</bool>
	 </property>
	</widget>
   </item>
   <item>
	<widget class="QToolButton" name="button_min">
	 <property name="text">
	  <string>...</string>
	 </property>
	 <property name="icon">
	  <iconset resource="cloud_disk.qrc">
	   <normaloff>:/image/Resources/image/images/title_min.png</normaloff>:/image/Resources/image/images/title_min.png</iconset>
	 </property>
	 <property name="iconSize">
	  <size>
	   <width>32</width>
	   <height>32</height>
	  </size>
	 </property>
	 <property name="autoRaise">
	  <bool>true</bool>
	 </property>
	</widget>
   </item>
   <item>
	<widget class="QToolButton" name="button_close">
	 <property name="text">
	  <string>...</string>
	 </property>
	 <property name="icon">
	  <iconset resource="cloud_disk.qrc">
	   <normaloff>:/image/Resources/image/images/login_close.png</normaloff>:/image/Resources/image/images/login_close.png</iconset>
	 </property>
	 <property name="iconSize">
	  <size>
	   <width>32</width>
	   <height>32</height>
	  </size>
	 </property>
	 <property name="autoRaise">
	  <bool>true</bool>
	 </property>
	</widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="cloud_disk.qrc"/>
 </resources>
 <connections/>
</ui>
